pagopa/pn-frontend
# Error when trying to open zendesk ticket with popup blocked - Fix/pn 10724

## Short description
When user browser has popups blocked, the POST of the zendesk form goes in error. So we have decided to open the form in the same page and show an error message when the POST going in error.

## List of changes proposed in this pull request
- Removed target _blank from ZendeskForm.tsx and managed error cases

## How to test
Disable popup in your browser -> login with pf -> click on assistance link -> fill the form and click on confirm -> check that the zendesk form is opened without errors
